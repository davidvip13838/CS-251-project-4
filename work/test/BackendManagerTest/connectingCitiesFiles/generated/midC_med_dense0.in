500
95 67
8 15
73 17
20 96
91 69
63 25
32 34
29 83
38 55
89 45
82 38
13 60
96 22
8 50
47 12
52 1
53 25
91 46
63 38
55 47
20 89
42 54
32 8
65 10
54 66
83 92
41 37
89 56
23 87
85 2
51 89
92 43
68 59
39 70
13 40
90 71
61 98
61 100
15 54
55 70
60 78
42 51
12 79
48 15
99 48
73 69
80 16
18 33
21 36
54 1
20 82
56 19
82 85
97 5
82 69
57 50
93 46
56 98
99 1
82 23
19 59
66 7
10 48
64 48
48 39
0 39
68 72
99 91
16 36
14 91
92 99
2 19
29 75
3 42
74 41
87 83
15 58
74 8
3 99
33 81
0 57
62 9
86 91
59 32
53 39
51 54
83 8
57 96
28 76
51 76
32 84
76 35
14 3
95 42
28 23
93 56
93 22
37 7
91 13
62 96
42 10
29 57
34 77
83 99
81 13
64 44
91 50
50 27
78 24
1 81
62 55
78 44
57 86
87 17
54 27
17 93
75 54
55 18
76 80
38 90
11 38
78 26
76 11
15 13
4 53
82 89
60 60
99 42
35 44
3 6
39 25
65 19
43 65
79 49
32 6
89 18
66 60
75 41
92 4
73 67
72 93
48 64
24 86
40 97
63 100
16 60
34 43
53 56
14 77
21 6
15 4
61 30
73 25
58 40
20 51
41 71
90 24
74 43
41 59
88 33
35 9
31 63
59 35
39 39
61 11
15 77
38 95
83 23
21 74
45 9
84 84
24 51
95 7
33 88
44 36
76 7
4 43
48 33
6 56
37 97
98 12
71 62
51 13
21 62
25 86
74 48
21 14
82 24
19 53
47 10
25 63
90 1
76 95
32 91
18 95
59 51
74 90
48 43
70 20
38 36
37 92
32 17
25 21
76 44
77 20
96 95
55 49
57 29
53 92
22 11
36 13
100 62
59 59
100 81
96 26
59 24
96 1
8 78
17 51
67 83
81 88
75 79
49 64
47 71
5 26
76 23
23 98
72 15
59 27
31 49
68 80
99 94
45 22
22 35
84 92
11 23
91 1
41 52
39 65
35 27
74 96
17 22
74 40
61 45
62 89
58 53
65 36
60 66
10 8
91 10
49 91
82 15
11 40
59 84
100 9
62 35
18 85
73 43
88 0
39 7
96 77
88 75
41 1
15 21
47 60
49 77
26 55
36 63
22 29
78 27
80 33
99 98
53 72
37 26
75 56
39 9
57 75
55 60
86 86
49 21
96 39
14 73
89 15
35 92
15 95
54 20
5 57
86 71
94 49
31 100
41 12
88 72
28 17
64 84
45 39
42 90
22 22
55 20
91 11
3 79
44 92
60 50
74 23
25 72
45 76
93 84
28 1
38 17
64 9
79 15
95 30
26 49
28 34
22 8
53 82
28 77
76 55
1 57
26 9
81 68
47 25
80 69
35 36
98 83
60 18
54 62
55 90
0 58
12 16
27 92
10 40
68 45
39 88
45 46
51 43
36 18
2 77
35 97
4 71
36 45
53 29
8 76
32 37
99 71
51 2
74 84
39 42
5 76
61 33
37 91
5 6
34 95
16 52
89 39
27 88
9 96
64 40
37 19
77 4
44 85
34 94
74 73
64 28
78 37
5 18
69 0
3 13
17 88
15 66
22 52
62 70
100 78
15 25
64 17
87 63
9 69
42 27
40 3
37 46
24 74
71 80
14 66
61 7
5 14
15 75
53 9
38 19
0 79
3 82
70 21
100 11
65 27
52 73
34 100
72 87
46 76
87 62
49 2
57 76
71 13
72 97
86 45
38 4
33 100
4 83
4 93
31 53
35 14
55 52
82 75
64 81
38 37
98 53
59 100
11 2
5 56
6 3
51 37
37 85
66 38
44 10
8 7
82 43
39 81
69 42
91 21
9 55
29 67
91 92
15 32
8 21
71 36
46 66
73 74
94 34
62 27
65 73
100 18
4 70
74 95
95 65
55 13
98 58
41 96
11 88
2 18
70 3
8 13
78 22
5 23
66 65
69 78
82 33
23 51
15 19
39 66
83 85
93 44
64 67
85 25
13 39
19 67
79 21
48 10
86 44
84 71
14 49
78 48
17 24
24 33
34 59
1 89
81 23
59 19
1 24
7 0
46 88
6 78
71 64
74 51
4 33
100 58
5 45
69 65
16 88
86 49
92 63
0 55
78 34
56 2
39 79
5 2
26 66
14 98
75 3
65 46
34 35
56 71
50 29
75 91
