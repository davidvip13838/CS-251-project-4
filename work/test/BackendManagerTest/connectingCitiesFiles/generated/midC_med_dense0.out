1523.9445619862147